pom.xml

```diff
@@ -34,6 +34,16 @@
       <artifactId>json</artifactId>
       <version>20200518</version>
     </dependency>
+    <dependency>
+      <groupId>org.jsoup</groupId>
+      <artifactId>jsoup</artifactId>
+      <version>1.11.3</version>
+    </dependency>
+    <dependency>
+      <groupId>com.google.code.gson</groupId>
```

Comment:
GSON technically isn't being used in this PR, but it'll end up being used in SearchServlet anyways

Reply:
None of the active PRs actually use it still...so I would defer until you actually do. No need to bundle this change with an unrelated PR.